### DISCUSSION TITLE
post/slack-angmondbot-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#94] [Announcements]
# post/slack-angmondbot-2/

분산 시스템 환경에서 슬랙봇 앙몬드를 운영하며 마주했던 트러블슈팅 경험에 대해 공유합니다.

https://tech.kakaopay.com/post/slack-angmondbot-2/

**Comment by yrnana** [comment]
오오... 프론트 개발자가 생각해보기 힘든 이슈들이네요
흥미롭게 잘 읽었습니다!
